### PR TITLE
28 provide docker agent image for sshlibrary

### PR DESCRIPTION
**Pull request by Maresther-B**
I can't succeed to create an "agents" subdirectory on dockerhub. I need to take more time to figure it out how doing this

### CODE REVIEW

File: .github/workflows/CI_agent_sshlibrary.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Publish docker sshlibrary agent image on dockerhub
+on:
+  push:
+
```
Comment (damies13) [inline .github/workflows/CI_agent_sshlibrary.yml:4]: Consider adding something like this to `push`:
```
    paths:
      - 'agent/rfswarm-sshlibrary/*'
```
See: [Running your workflow only when a push affects specific files](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-affects-specific-files)

You'll probably want something similar for the base image

**Review by damies13 — CHANGES_REQUESTED** [review]
have a look at my suggested change in the review

File: .github/workflows/CI_agent_sshlibrary.yml (hunk 1 above)
Comment (Maresther-B) [reply .github/workflows/CI_agent_sshlibrary.yml:4]: Hello Dave, take  for your reply. .K I'm having a look now. Ksenia made some big changes by adding tag name versioning and  build images only when changes are detected. So I'm going to check all of them at the same time.

Comment (Maresther-B) [reply .github/workflows/CI_agent_sshlibrary.yml:4]: Ok I just checked and Ksenia has already took your suggestion in account :-) . Thanks Dave

**Pull request merged**